modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,692 @@
+
+In {productname} 6.3, the user selection is now preserved when toggling a list to a checklist.
+
+For information on the **Checklist** plugin, see xref:checklist.adoc[Checklist].
+
+=== Comments 3.2.0
+
+The {productname} 6.3 release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** 3.2.0 includes one fix and one change:
+
+==== A comment could be inserted when in an empty editor or block
+
+In previous {productname} versions a bug existed that allowed comments to be placed into an empty editor or block.
+
+With this update, the comments button, menu-item, comment sidebar and shortcut keys are now disabled when the editor contains no content.
+
+==== The selection now expands to encompass the word containing the insertion point when a comment is added
+
+{productname} 6.3 adds xref:additions[two new API functions] — `tinymce.selection` and `tinymce.dom.RangeUtils`.
+
+With these two functions available, if there is no selected text when a new comment is added, {productname} automatically extends the selection to encompass the word containing the insertion point and links the newly added comment to this automatically selected word.
+
+For information on the **Comments** plugin, see: xref:introduction-to-tiny-comments.adoc[Introduction to Tiny Comments].
+
+
+=== Merge Tags 1.1.0
+
+The {productname} 6.3 release includes an accompanying release of the **Merge Tags** premium plugin.
+
+**Merge Tags** 1.1.0 includes one addition, one fix, and one change:
+
+==== New mceMergeTag command to wrap selected content in a merge tag
+
+A new command, `mceMergeTag` has been added which can convert any string into a merge tag. This new command is available to use when defining xref:content-behavior-options.adoc#text_patterns[Text Patterns].
+
+Previously only a predefined list of merge tags could be used. When the new `mceMergeTag` command is added, any valid Merge Tag string can be converted into a merge tag.
+
+As well, configuring the inline `text_pattern` with this command allows merge tags to be entered by manual typing.
+
+Type the prefix, some content, the suffix, and then a word boundary character, and the `mceMergeTag` command will immediately wrap that content in a complete merge tag.
+
+In the following example, the Merge Tag prefix and suffix are the default, `{{` and `}}`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea', // change this value according to your HTML
+  plugins: 'mergetags',
+  text_patterns: [
+    { start: '{{', end: '}}', cmd: 'mceMergeTag' }
+  ],
+  mergetags_list: [
+    {
+      value: 'merge.tag',
+      title: 'A merge tag.'
+    },
+  ],
+  toolbar: 'mergetags',
+});
+----
+
+In this example, typing `{{string}}` followed by a word boundary character (such as a space, full-stop, question mark or comma) will result in the _string_ being turned into an active merge tag.
```

Comment:
I would be much better if this line is above the example

Reply:
Respectfully disagree here. The current structure is

1. Intro to example and example setup
2. Example itself
3. Using the example.

Pushing _Using the example_ above _Example itself_ is, to me, putting the cart before the horse.